deb/publish.go

```diff
@@ -485,9 +485,49 @@ func (p *PublishedRepo) GetLabel() string {
 	return p.Label
 }
 
+// GetAddonFiles returns a map of files to be added to a repo. Key being the relative
+// path from component folder, and value being the full local FS path.
+func (p *PublishedRepo) GetAddonFiles(addonDir string, component string) (map[string]string, error) {
+	files := make(map[string]string)
+
+	if addonDir == "" {
+		return files, nil
+	}
+
+	fsPath := filepath.Join(addonDir, p.Prefix, "dists", p.Distribution, component)
+	if err := filepath.Walk(fsPath, func(path string, info os.FileInfo, err error) error {
```

Comment:
If we use Walk here I think it also needs to implement some level of symlink handling. I can entirely imagine people wanting to keep the data in some other dir and then `ln -sf` resources it into the addon dir as they get updated.